src/features/payment/api/index.ts

```diff
@@ -0,0 +1,71 @@
+import { queryClient } from "@/lib/query";
+import { type User, updateUser } from "@/models/users";
+import type { Stripe, StripeElements } from "@stripe/stripe-js";
+import { useMutation } from "@tanstack/react-query";
+import { useSuspenseQuery } from "@tanstack/react-query";
+
+const addThanks = async ({
+  documentId,
+  user,
+  thanks,
+}: {
+  documentId: string;
+  user: User;
+  thanks: number;
+}) => {
+  updateUser({
+    documentId,
+    user: { thanks: user.thanks + thanks },
+  });
+};
+
+type StripeIntent = {
+  client_secret: string;
+};
+
+const postStripeIntent = async (): Promise<StripeIntent> => {
+  const res = await fetch(
+    "https://api.stripe.com/v1/payment_intents?amount=1099&currency=usd",
```

Comment:
ここのパラメータってないとまずいですか？ダミーの値に見えるので

```suggestion
    "https://api.stripe.com/v1/payment_intents",
```

Reply:
payment intentを生成するのにamountとcurrencyの指定が必須です。
おそらく、payment intentを生成するタイミングが間違っているのですが、解決案が見つからずこうしています。
いったんはコメントを残す形で対応させていただきます 🙇